src/app/fundraisings/[fundraisingId]/jars/Statistics/StatisticsSection/StatisticsSection.module.css

```diff
@@ -34,6 +34,8 @@
   height: 30px;
   margin-right: 15px;
   max-height: 300px;
+  border: 1px solid black;
```

Comment:
You sure `1px` is enough? Looks like very thin border if I want to see color

Reply:
Ah., it's for bars. I dunno 🤔 Matter of taste I suppose.

Maybe not black? Something little bit lighter? Just to have a border but not draw an attention?

Reply:
It is black so that the light progress bars are easily visible on the light background, also I wanted to make it in the same style as the existing current progress bar(the same border color, thickness, and roundness).

<img width="509" alt="Screenshot 2024-02-20 at 21 56 15" src="https://github.com/zonzujiro/korchivnyky-dashboard/assets/64706187/d6761564-4025-4850-a1c6-6a5590ba0fb9">